### PR TITLE
Adds Payments Processing Specialist job ad

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jesssummerfield , @bamesjerry : please review.

### CODE REVIEW

File: app/pages/about/jobs/positions/payments-processing-specialist.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The ideal candidate</h4>
+      <ul className='list u-margin-Tm'>
+        <li>
+        You have great analytical skills - you’re an expert on Excel and curious about data.
+        </li>
+        <li>
+        You love to analyse processes and constantly looks for ways to improve them.
+        </li>
+        <li>
+        You have technical degree.
```
Comment (timrogers) [inline app/pages/about/jobs/positions/payments-processing-specialist.js]: a technical degree

**Comment by gabrielamatias** [comment]
corrected the bullet point, thanks!

**Comment by bamesjerry** [comment]
:+1: (with @jesssummerfield 's changes)

**Comment by gabrielamatias** [comment]
@jesssummerfield , @bamesjerry : please review.

**Pull request merged**